client/my-sites/checkout/controller.jsx

```diff
@@ -42,7 +42,7 @@ import { getProductSlugFromContext } from './utils';
 
 const debug = debugFactory( 'calypso:checkout-controller' );
 
-export function checkoutSiteless( context, next ) {
+export function checkoutJetpackSiteless( context, next ) {
```

Comment:
I just wanted to confirm that this workflow doesn't need to support `isGiftPurchase`

Reply:
To my knowledge, it does not. I asked in Slack (p1667512942661609-slack-C0117V2PCAE) and folks said that gift purchases require being logged-in.

Reply:
Yeah, looking through this again, I don't think you could arrive here through a gift route.